.github/workflows/uncrustify.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         run: |
           uncrustify --version
           find . -not -path '.*catch_assert.h' -iname "*.[hc]" -exec uncrustify -c tools/uncrustify.cfg --no-backup --replace {} +
+          find . -not -path '.*catch_assert.h' -iname "*.[hc]" -exec uncrustify -c tools/uncrustify.cfg --no-backup --replace {} +
```

Comment:
More of a suggestion for future work...It would be cool to run uncrustify as long as it is modifying files. I'd think this could be done in a loop by simply staging before every uncrustify operation and then after uncrustify-ing checking if any modified files are present.

Reply:
Agreed. I am also thinking of just uncrustify-ing the files which were actually changed by the PR. However, these optimizations can be added incrementally.